packages/shared/src/hooks/useProfileTooltip.ts

```diff
@@ -57,12 +57,9 @@ export const useProfileTooltip = ({
     }
   }, [data, error]);
 
-  return useMemo(
-    () => ({
-      data,
-      isLoading: isPending,
-      fetchInfo: () => setShouldFetch(true),
-    }),
-    [data, isPending, setShouldFetch],
-  );
+  return {
```

Comment:
Shouldn't we need the memo here for data changes still?

Reply:
not really, you never use [the whole object](https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/248020993/Guidelines#Other-custom-hooks:~:text=We%20do%20not%20memoize%20return%20object%20because%20usage%20of%20our%20business%20logic%20hooks%20should%20be%20through%20destructuring%20assignment.) returned from hook

Reply:
I've checked the usage, and useProfileTooltip is only used here, so this change is quite safe, imo.
`data` and `isPending` are handled/cached by react-query, we should care only about `setShouldFetch`, which shouldn't cause issues 👀